applications/StructuralMechanicsApplication/tests/test_constitutive_law.py

```diff
@@ -733,5 +781,29 @@ def __init__(self):
     def create_constitutive_Law():
         return StructuralMechanicsApplication.LinearJ2PlasticityPlaneStrain2DLaw()
 
+class LinearIsotropicDamage(LinearElastic):
+    def __init__(self):
+        self.young_modulus = 3000
+        self.poisson_ratio = 0.3
+        self.yield_stress = 2.0
+        self.infinity_yield_stress = 3.0
+        self.isotropic_hardening_modulus = 0.3
+
+    def create_properties(self, model_part):
+        properties = LinearElastic.create_properties(self, model_part)
+        properties.SetValue(KratosMultiphysics.YIELD_STRESS, self.yield_stress)
+        properties.SetValue(StructuralMechanicsApplication.INFINITY_YIELD_STRESS, self.infinity_yield_stress)
+        properties.SetValue(KratosMultiphysics.ISOTROPIC_HARDENING_MODULUS, self.isotropic_hardening_modulus)
+        return properties
+
+class LinearIsotropicDamage3D(LinearIsotropicDamage):
+    def __init__(self):
+        LinearIsotropicDamage.__init__(self)
+        self.dim = 3
+
+    @staticmethod
+    def create_constitutive_Law():
+        return StructuralMechanicsApplication.LinearIsotropicDamage3DLaw()
```

Comment:
I would say that this should be a general method, or you change the name

Reply:
The CL tests are based on the recent code reorganization by @PhilippeBussetta . I'd like to discuss it on a separate issue.